misk-hibernate-testing/src/main/kotlin/misk/jdbc/ScaleSafetyChecker.kt

```diff
@@ -0,0 +1,121 @@
+package misk.jdbc
+
+import com.squareup.moshi.Moshi
+import misk.moshi.adapter
+import misk.okio.split
+import mu.KotlinLogging
+import okio.BufferedSource
+import okio.ByteString.Companion.encodeUtf8
+import java.io.EOFException
+import java.sql.Connection
+import java.sql.SQLException
+import java.sql.SQLSyntaxErrorException
+import java.sql.Timestamp
+
+object ScaleSafetyChecker {
```

Comment:
nit: `Checker` makes it sound like it is something but this is really just a library so maybe just call it `ScaleSafetyChecks`